app/src/main/res/values-sv/strings.xml

```diff
@@ -51,42 +51,42 @@
     <string name="swap_prev_next">Växla hörlurars föregående- och nästaknappar</string>
 
     <!-- FAQ -->
-    <string name="faq_1_title">How can I fast-forward songs?</string>
-    <string name="faq_1_text">You can click on the current or max duration texts near the seekbar, that will move the song either backward, or forward.</string>
+    <string name="faq_1_title">Hur kan jag snabbspola låtar?</string>
+    <string name="faq_1_text">Du kan klicka på låtens nuvarande tid eller längd nära sökfältet, vilket kommer att spola låten antingen bakåt eller framåt.</string>
 
     <!-- Strings displayed only on Google Playstore. Optional, but good to have -->
     <!-- App title has to have less than 50 characters. If you cannot squeeze it, just remove a part of it -->
-    <string name="app_title">Simple Music Player - A cool player for your songs</string>
+    <string name="app_title">Simple Music Player - En cool spelare för dina låtar</string>
     <!-- Short description has to have less than 80 chars -->
-    <string name="app_short_description">A clean music player with a customizable widget, stylish interface and no ads.</string>
+    <string name="app_short_description">En ren musikspelare med en anpassningsbar gränssnittskomponent, snyggt gränssnitt och ingen reklam.</string>
```

Comment:
Make the short app description shorter please, it is currently 92 chars long. The max can be 80. Th eapp title is 52, can be max 50. Thanks